### ISSUE TITLE
note directive causes man page writer to throw exception

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1214]
```
Sphinx version: 1.1.3
Python version: 2.7.2
Docutils version: 0.10 release
Jinja2 version: 2.7-dev
```

In my document I have a note directive. Generating html works as expected. Generating man pages causes a stack trace.

```
File "docutils/writers/manpage.py", line 411, in depart_admonition
  self.depart_block_quote()
File "docutils/writers/manpage.py", line 449, in depart_block_quote
  self.dedent()
File "docutils/writers/manpage.py", line 865, in dedent
  self._indent.pop()
IndexError: pop from empty list
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1214
- Originally reported by: Anonymous
- Originally created at: 2013-07-16T23:46:57.445

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-18 14:12:44+00:00_

Can you attach your rst file (or smallest example that can reproduce the issue) ?

**Comment by shimizukawa** [comment]
_From Thatcher Peskens on 2013-07-25 23:56:42+00:00_

Just ran into this issue as well.

All I need is 

.. note:: 
    some text

**Comment by shimizukawa** [comment]
_From Thatcher Peskens on 2013-07-26 00:01:20+00:00_

Here is my file. I modified an existing file, that is why the date is old. The version I am using is Sphinx v1.1.3

```
#!python

.. TeStING documentation master file, created by
   sphinx-quickstart on Mon May 20 16:58:28 2013.
   You can adapt this file completely to your liking, but it should at least
   contain the root `toctree` directive.

Welcome to TeStING's documentation!
===================================

Contents:

.. toctree::
   :maxdepth: 2



Indices and tables
==================

* :ref:`genindex`
* :ref:`modindex`
* :ref:`search`

.. note::
    some text

```

run `make man`

**Comment by shimizukawa** [comment]
_From Thatcher Peskens on 2013-07-26 00:23:44+00:00_

Ok, I can actually confirm this issue has been resolved in 1.2b1.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-26 01:57:04+00:00_

[Thatcher Peskens](https://bitbucket.org/dhrp) OK. Thanks for your confirmation!

I think #998 is a same issue. May I close this?
